grails-app/controllers/org/broadinstitute/orsp/ConsentGroupController.groovy

```diff
@@ -129,6 +129,7 @@ class ConsentGroupController extends AuthenticatedController {
             queryService.updateCollectionLinkStatus(params.consentKey, params.projectKey, params.type == "unlink" ? CollectionLinkStatus.UNLINKED.name : CollectionLinkStatus.REJECTED.name)
             List<ConsentCollectionLink> links = queryService.findConsentCollectionLinksByProjectKeyAndConsentKey(params.projectKey, params.consentKey)
             deleteCollectionLinks(links)
+            notifyService.sendApproveRejectLinkNotification(params.projectKey.toString(), params.consentKey.toString(), false)
```

Comment:
Nit (here and throughout): `params.projectKey.toString()` can NPE and could use null-safe traversal.